packages/typescript/ai-elevenlabs/src/realtime/types.ts

```
@@ -17,14 +17,21 @@ export interface ElevenLabsRealtimeTokenOptions {
   }
 }
 
+import type { DebugOption } from '@tanstack/ai'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)